frontend/src/metabase/admin/settings/selectors.js

```diff
@@ -428,86 +427,21 @@ export const ADMIN_SETTINGS_SECTIONS = {
     settings: [],
   },
   "embedding-in-other-applications/standalone": {
-    settings: [
-      {
-        key: "-breadcrumb",
-        widget: () => {
-          return (
-            <Breadcrumbs
-              size="large"
-              crumbs={[
-                [
-                  t`Embedding`,
-                  "/admin/settings/embedding-in-other-applications",
-                ],
-                [t`Static embedding`],
-              ]}
-            />
-          );
-        },
-      },
-      {
-        key: "embedding-secret-key",
-        display_name: t`Embedding secret key`,
-        description: t`Standalone Embed Secret Key used to sign JSON Web Tokens for requests to /api/embed endpoints. This lets you create a secure environment limited to specific users or organizations.`,
-        widget: SecretKeyWidget,
-        getHidden: (_, derivedSettings) => !derivedSettings["enable-embedding"],
-        props: {
-          confirmation: {
-            header: t`Regenerate embedding key?`,
-            dialog: t`This will cause existing embeds to stop working until they are updated with the new key.`,
-          },
-        },
-      },
-
-      {
-        key: "-embedded-resources",
-        display_name: t`Manage embeds`,
-
-        widget: EmbeddedResources,
-        getHidden: (_, derivedSettings) => !derivedSettings["enable-embedding"],
-      },
-      {
-        key: "-redirect-widget",
-        widget: () => (
-          <RedirectWidget to="/admin/settings/embedding-in-other-applications" />
-        ),
-        getHidden: (_, derivedSettings) => derivedSettings["enable-embedding"],
-      },
-    ],
+    component: StaticEmbeddingSettings,
+    settings: [],
   },
   "embedding-in-other-applications/sdk": {
     component: EmbeddingSdkSettings,
     settings: [],
   },
   "embedding-in-other-applications/full-app": {
-    settings: [
-      {
-        key: "-breadcrumbs",
-        widget: () => {
-          return (
-            <Breadcrumbs
-              size="large"
-              crumbs={[
-                [
-                  t`Embedding`,
-                  "/admin/settings/embedding-in-other-applications",
-                ],
-                [t`Interactive embedding`],
-              ]}
-            />
-          );
-        },
-      },
-      {
-        key: "-redirect-widget",
-        widget: () => (
-          <RedirectWidget to="/admin/settings/embedding-in-other-applications" />
-        ),
-        getHidden: (_, derivedSettings) =>
-          PLUGIN_EMBEDDING.isEnabled() && derivedSettings["enable-embedding"],
-      },
-    ],
+    // We need to do this because EE plugins would load after this file in unit tests
+    component: ({ updateSettings }) => (
```

Comment:
unfortunately when testing in Jest, we still intialize EE plugin last in some cases if not all. In this case, `SettingsEditor` tests will import `selectors` before initialize EE plugins, so it will fail if we just use.
```ts
component: PLUGIN_ADMIN_SETTINGS.InteractiveEmbeddingSettings
```

Although that actually works in the app.